rasa/telemetry.py

```diff
@@ -573,6 +573,18 @@ def strip_sensitive_data_from_sentry_event(
     for value in event.get("exception", {}).get("values", []):
         for frame in value.get("stacktrace", {}).get("frames", []):
             frame["abs_path"] = ""
+
+            if "site-packages" in frame["filename"]:
+                # drop site-packages and following slash / backslash
+                frame["filename"] = frame["filename"].split("site-packages")[-1][1:]
+            if "dist-packages" in frame["filename"]:
+                # drop dist-packages and following slash / backslash
+                frame["filename"] = frame["filename"].split("dist-packages")[-1][1:]
```

Comment:
Do you think it would still be valuable to keep `dist-packages/` and `site-packages/` in the filename? Could it influence Sentry's "App Only / Full" toggle behaviour?

Reply:
mhm that is a good point, yes it actually might :+1:

Reply:
yes that is a good idea :+1: